### PR TITLE
add issue and PR templates

### PR DESCRIPTION
**Pull request by classabbyamp**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Report a bug to help us improve qrm
```
Comment (classabbyamp) [inline .github/ISSUE_TEMPLATE/bug_report.md]: genericise

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+1. Run command '...' with input '...'
+2. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System (include if related to running the bot):**
```
Comment (classabbyamp) [inline .github/ISSUE_TEMPLATE/bug_report.md]: genericise

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for qrm
```
Comment (classabbyamp) [inline .github/ISSUE_TEMPLATE/feature_request.md]: genericise

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Report a bug to help us improve qrm
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
```
Comment (classabbyamp) [inline .github/ISSUE_TEMPLATE/bug_report.md]: all descriptions to html comments

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Run command '...' with input '...'
```
Comment (classabbyamp) [inline .github/ISSUE_TEMPLATE/bug_report.md]: "clear steps to reproduce bug"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+2. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```
Comment (classabbyamp) [inline .github/ISSUE_TEMPLATE/bug_report.md]: and logs etc

Comment (classabbyamp) [reply .github/ISSUE_TEMPLATE/bug_report.md]: make last

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (classabbyamp) [reply .github/ISSUE_TEMPLATE/bug_report.md]: "environment"

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (classabbyamp) [inline .github/ISSUE_TEMPLATE/feature_request.md]: add newlines here, convert to headers

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+Fixes #{issue}
+
+### Type of change
+
+Please delete options that are not relevant.
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- This change requires a documentation update
```
Comment (classabbyamp) [inline .github/pull_request_template.md:16]: add major/minor refactor

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
```
Comment (classabbyamp) [inline .github/ISSUE_TEMPLATE/feature_request.md]: rewrite

File: .github/pull_request_template.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
+### Description
+
+*Describe the changes you made here.*
+
+Fixes #{issue}
```
Comment (classabbyamp) [inline .github/pull_request_template.md:5]: add comment about adding more issues

File: .github/pull_request_template.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+
+### Type of change
+
+Please delete options that are not relevant.
```
Comment (classabbyamp) [inline .github/pull_request_template.md]: comment

File: .github/pull_request_template.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+
+*Describe the procedure used for verifying your changes here.*
+
+### Checklist
+
+- [ ] Issue exists for PR
+- [ ] Code reviewed by the author
+- [ ] Code documented (comments or other documentation)
+- [ ] Changes tested
+- [ ] All tests pass (see `DEVELOPING.md`)
```
Comment (classabbyamp) [inline .github/pull_request_template.md]: if exists

File: .github/pull_request_template.md [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] `CHANGELOG.md` updated
```
Comment (classabbyamp) [inline .github/pull_request_template.md]: if needed

**Review by classabbyamp — COMMENTED** [review]
comments from @0x5c

**Comment by classabbyamp** [comment]
also, link mentioned files (rel links)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+### Expected behavior
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+### Environment
+
+<!-- include if relevant -->
+
+<!-- e.g. Linux, Docker -->
+- OS:
```
Comment (classabbyamp) [inline .github/ISSUE_TEMPLATE/bug_report.md]: change to one commented list item `- thing version`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (classabbyamp) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: fix caps throughout all

**Review by classabbyamp — COMMENTED** [review]
comments from @0x5c

**Review by 0x5c — APPROVED** [review]

**Pull request merged**